templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig

```diff
@@ -31,7 +31,8 @@
           {% endif %}
           {% if primary_type == 'document' %}
             {% if content.field_primary_action_document|render|striptags|trim is not empty %}
-              {{ content.field_primary_action_document }}
+              <a href="{{ file_url(content.field_primary_action_document[0]['#media'].field_media_file.entity.uri.value) }}" class="standalone-link button" download title="{{ content.field_primary_action_title|render|striptags|trim }}">{{ content.field_primary_action_title|render|striptags|trim }}</a>
```

Comment:
There is a preprocessor for the CTA paragraph.
Better to extract all the values there and add them as extra theme variables.
Makes it easier to print the variables in the template ;-)

Reply:
Juist ja :) toegepast voor de download links.